src/Functions/if.cpp

```diff
@@ -76,75 +76,17 @@ inline void fillVectorVector(const ArrayCond & cond, const ArrayA & a, const Arr
 {
 
     size_t size = cond.size();
-    bool a_is_short = a.size() < size;
```

Comment:
There is similar logic in multiIf, maybe it should be adjusted too.

Reply:
:+1: